alerts/KubeJobFailed.md

```diff
@@ -0,0 +1,55 @@
+# KubeJobFailed
+
+## Meaning
+
+[This alert][KubeJobFailed] is triggered for the case that the number of job
+execution attempts exceeds the `backoffLimit`. A job can therefore create one or
+many pods for its tasks.
+
+## Impact
+
+A task has not finished correctly. Depending on the task, this has a different
+impact.
+
+## Diagnosis
+
+The alert should contain the job name and the namespace where that job failed.
+Follow the particular mitigation steps according to that. For example:
+
+```text
+ - alertname = KubeJobFailed
+... 
+ - message = Job openshift-logging/elasticsearch-delete-app-1600855200 failed to complete.
+```
+
+## Mitigation
+
+### Failed `elasticsearch-*` jobs in `openshift-logging` namespace
```

Comment:
label this as an `example`

Reply:
I think we cant keep it as just an example since in the following the indexManagement Pods are queried, which should be pretty specific to elastisearch. 
`$ oc get pod -l component=indexManagement`
We could just drop that and add the generic methodology instead.

Reply:
sounds good